sn_auditor/src/dag_db.rs

```diff
@@ -249,41 +250,62 @@ impl SpendDagDb {
                 continue;
             }
 
-            // get a copy of the current DAG
-            let mut dag = { self.dag.clone().read().await.clone() };
+            let new_utxos = if cfg!(feature = "dag-collection") {
+                self.crawl_and_generate_local_dag(
```

Comment:
the `tracking_performance` is only got updated within `beta_background_process_spend`,
which seems now out of sync with the actually background crawling track ?
will need to move the tracking to the correct place so that proper performance comparision can be undertaken.